crates/frontend/src/pages/context_override.rs

```diff
@@ -335,20 +335,35 @@ pub fn context_override() -> impl IntoView {
                                 (context.clone(), overrides)
                             })
                             .collect::<Vec<(Context, Map<String, Value>)>>();
-                        ctx_n_overrides
-                            .into_iter()
-                            .map(|(context, overrides)| {
-                                view! {
-                                    <ContextCard
-                                        context=context
-                                        overrides=overrides
-                                        handle_edit=handle_context_edit
-                                        handle_clone=handle_context_clone
-                                        handle_delete=handle_context_delete
-                                    />
-                                }
-                            })
-                            .collect_view()
+                        if ctx_n_overrides.is_empty() {
```

Comment:
You can use `<Show>` component to achieve this.
And the else branch in not needed I guess, move the code out of the else branch.

Reply:
@ShubhranshuSanjeev can we take this later? I think this is fine for now.